### DISCUSSION TITLE
eZ Navi-Delate v2.0a completed!

### DISCUSSION THREAD

**Discussion by radensb** [radensb/eZ_Navi-Delete#5] [General]
For those interested in the stats of new hardware, I have (finally) completed v2.0a and sent the design files to MFG for quote. I should know more about pricing and availability of new hardware by next week.

v2.0a is functionally identical to the Bluetooth version of v1.0a. The major revision change is due to changing the connector family from TE Connectivity to Molex since premade Molex pigtails can be easily purchased which greatly simplifies the wiring for installation. The pigtails eliminate the need to use special crimping tools to crimp the connector contacts to the wires. Additionally, some ESD protection diodes were added to the design for good measure.

Accompanying the new connector design is a PCB adapter kit for the factory harness. This allows for an easier interface to the car. I will be adding details to with Wiki soon.

**Comment by Ghostvette** [comment]
I'm looking forward to seeing the new set up.

**Reply by radensb** [reply]
FYI - I have put some time into updating the Wiki with the v2.0a stuff, including new wiring options. Check it out!
